lib/api/_loaders/command.js

```diff
@@ -204,6 +204,7 @@ class CommandLoader extends BaseCommandLoader {
 
     this.validateMethod(parent);
 
+    const redact = this.module.RedactParams || false;
```

Comment:
what is this doing?

Reply:
it takes a value from static getter from the command

Reply:
https://github.com/nightwatchjs/nightwatch/pull/2672/files#diff-ea3b757539fedd449edfa8d90350151f7dd528abf5b0266f02009ca2b96d136fR37

Reply:
ah yes, missed that.